corehq/apps/auditcare/tasks.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import sys
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+from couchdbkit.ext.django.loading import get_db
+
+from dimagi.utils.couch.database import iter_docs
+from dimagi.utils.dates import force_to_datetime
+from dimagi.utils.logging import notify_exception
+
+from corehq.apps.auditcare.models import (
+    ACCESS_LOGIN,
+    ACCESS_LOGOUT,
+    ACCESS_FAILED,
+    AccessAudit,
+    NavigationEventAudit,
+)
+from corehq.apps.auditcare.utils.export import get_sql_start_date
+from corehq.util.soft_assert import soft_assert
+
+log = logging.getLogger(__name__)
+
+ACCESS_LOOKUP = {
+    "login": ACCESS_LOGIN,
+    "logout": ACCESS_LOGOUT,
+    "login_failed": ACCESS_FAILED,
+}
+
+
+@periodic_task(run_every=crontab(minute="30"), queue='background_queue')
+def copy_events_to_sql(limit=1000):
+    db = get_db("auditcare")
+    start_date = get_sql_start_date()
+    log.info(f"Initial start date: {start_date}")
+
+    startkey = [
+        start_date.year,
+        start_date.month,
+        start_date.day,
+        start_date.hour,
+        start_date.minute,
+        start_date.second,
+        start_date.microsecond,
```

Comment:
I'm assuming there won't be any duplicate records at the microsecond level.

Reply:
Storing the couch id in SQL sounds reasonable, it's worked well for other migrations.

Reply:
Added couch id and duplication check in bbc30df77f24da7fd0c3677637200c6898c308e2